src/cr/cube/matrix.py

```diff
@@ -725,6 +821,10 @@ def _column_generator(self):
             self.table_margin.T,
         )
 
+    @lazyproperty
+    def _pairwise_overlap_total(self):
```

Comment:
```suggestion
    def _pairwise_overlap_total(self):
        return np.sum(np.sum(self._counts, axis=1), axis=2)
        """
        Given a 4d hypercube of multiple response items, return the
        symmetric square matrix of valid observations between all pairs.
        n1 = 2; n2 = 2; n12 = 1; overlap total = 3
        """
```

Reply:
Docstrings should be of the following PEP 257 form:

```python
def method(self):
    """Return 2D ndarray symmetric-square matrix of valid observations.

    Given a 4D hypercube of multiple-response (MR) items, this method calculates the
    *symmetric square matrix* which is a fancy word for {this simple idea}.
    """
```

The key features are a single line to start, short enough not to wrap, that gives some gist of what to expect but should always start by mentioning the return-type, since that is the effective "type" of the function. Then, when more is called for, a separate passage, separated by a blank line that fills in any important remaining details. The key thing is to describe the contract of the method, i.e. what it gives back and what you need to give it, in that order.